index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/b33cf67da8.js" crossorigin="anonymous"></script>    
+    <title>My First Portfolio</title>
+</head>
+<body>
+        <header class="welcome-container">
+            <a class="welcome">Welcome</a>
+            <a class="menu-icon"><i class="ri-menu-line"></i></a>
+        </header>
+        <section class="greetingsandsocial">
+            <h2 class="intro1">Hey there. I'm Edgard</h2>
+            <h2 class="intro2">I'm a software developer</h2>
+            <p class="intromessage"> I can help you build a product , feature or website. 
+            Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.</p>
+                <ul class="social-container">
+                    <li><a href="https://twitter.com/soza_edgard"><i class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="https://www.linkedin.com/in/edgard-soza-sobalvarro/"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-m"></i></a></li>
+                    <li><a href="https://github.com/edgardsoza"><i class="fa-brands fa-github"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+                </ul>
+        </section>
+    <section class="recent">
+        <img class="disabled" src="images/disabled.png" alt="">
+        <h1 class="recent-works">My Recent Works</h1>
+        <a><i class="fa-solid fa-horizontal-rule"></i></a>
```

Comment:
:orange_circle: [OPTIONAL]  The `<a>` tag defines a hyperlink, which is used to link from one page to another. As you can see there is no need to use `<a>` tag here you should rather use a` <div>`tag. Kindly replace the a tag with a div tag

Reply:
@bienvenuushindi done!!